### PR TITLE
docs: fix linter errors for property-binding.md

### PR DESCRIPTION
**Pull request by dsnoeijer**
docs: fix linter errors for property-binding.md 2

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #47179

## What is the new behavior?
No more linter errors in the file.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/property-binding.md [hunk 1]
```diff
@@ -85,9 +85,9 @@ To set the model property of a custom component for parent and child components
 <code-example path="property-binding/src/app/app.component.html" region="model-property-binding" header="src/app/app.component.html"></code-example>
 
 
-## Toggling button functionality
+## Toggling button features
 
-To disable a button's functionality depending on a Boolean value, bind the DOM `disabled` property to a property in the class that is `true` or `false`.
+To turn off a button's features depending on a Boolean value, bind the DOM `disabled` property to a property in the class that is `true` or `false`.
```
Comment (pkozlowski-opensource) [inline aio/content/guide/property-binding.md:90]: The actual DOM property is named [disabled](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled) so renaming it to "turn off" will be rather confusing IMO.

File: aio/content/guide/property-binding.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ In most cases, the target name is the name of a property, even when it appears t
 
 In this example, `src` is the name of the `<img>` element property.
 
-The brackets, `[]`, cause Angular to evaluate the right-hand side of the assignment as a dynamic expression.
+The brackets, `[]`, cause Angular to resolve the right-hand side of the assignment as a dynamic expression.
```
Comment (jelbourn) [inline aio/content/guide/property-binding.md:43]: "Evaluate" is the more technically accurate term here

File: aio/content/guide/property-binding.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ Omitting the brackets renders the string `parentItem`, not the value of `parentI
 
 ## Setting an element property to a component property value
 
-To bind the `src` property of an `<img>` element to a component's property, place the target, `src`, in square brackets followed by an equal sign and then the property.
+You can bind the `src` property of an `<img>` element to a component's property. This is done by placing the target, `src`, in square brackets followed by an equal sign and then the property.
```
Comment (jelbourn) [inline aio/content/guide/property-binding.md:55]: This proposal is replacing active voice with passive voice

File: aio/content/guide/property-binding.md [hunk 4]
```diff
@@ -72,7 +72,7 @@ To use property binding using `colSpan`, type the following:
 
 <code-example path="attribute-binding/src/app/app.component.html" region="colSpan" header="src/app/app.component.html"></code-example>
 
-To disable a button when the component says that it `isUnchanged`, type the following:
+To turn off a button when the component says that it `isUnchanged`, type the following:
```
Comment (jelbourn) [inline aio/content/guide/property-binding.md:75]: `disable` is the more technically accurate term

Comment (bob-watson) [reply aio/content/guide/property-binding.md:75]: @jelbourn would `deactivate` work as an inclusive and accurate substitute for `disable`?

**Comment by dsnoeijer** [comment]
@jelbourn @bob-watson 
I agree with the comments about disable and evaluate, however they give a Google_WordListSuggestions warning. I can disable the vale style rule for these if that is the preferred option.

File: aio/content/guide/property-binding.md (hunk 2 above)
Comment (bob-watson) [reply aio/content/guide/property-binding.md:43]: Jeremy's correct.

We might need to reconsider removing `evaluate` from the substitution list. For now, please revert this change. 

If you want to squelch the linter message, you can flag it as an exception with the comments in this example.

```suggestion
<!-- vale Angular.Google_WordListSuggestions = NO -->

The brackets, `[]`, cause Angular to evaluate the right-hand side of the assignment as a dynamic expression.

<!-- vale Angular.Google_WordListSuggestions = NO -->
```

You can use these comments around the other exceptions below, as well.

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for the edits.
Please review the comments in the file.
Some of your edits identified some changes we might need to make to our linter. In the meantime, they'll need to be reverted. I put more information in the file comments.

Thanks for your effort and your patience.

**Comment by dsnoeijer** [comment]
I added a fixup commit with your suggestions. If it's all good i will squash it.

File: aio/content/guide/property-binding.md (hunk 4 above)
Comment (jelbourn) [reply aio/content/guide/property-binding.md]: Not in this case. When we're talking about HTML controls, `disable` is the canonical term: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#disabled
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

**Review by bob-watson — APPROVED** [review]
LGTM
This just needs a squash. Make sure the resulting commit message starts with `docs: `.
Thanks!

reviewed-for: global-docs-approvers

**Comment by dsnoeijer** [comment]
@bob-watson Done.

File: aio/content/guide/property-binding.md [hunk 5]
```diff
@@ -84,10 +88,13 @@ To set the model property of a custom component for parent and child components
 
 <code-example path="property-binding/src/app/app.component.html" region="model-property-binding" header="src/app/app.component.html"></code-example>
 
+## Toggling button features
+
+<!-- vale Angular.Google_WordListSuggestions = NO -->
 
-## Toggling button functionality
+To use a Boolean value to disable a button's features, bind the `disabled` DOM attribute to a Boolean property in the class.
```
Comment (jelbourn) [inline aio/content/guide/property-binding.md:95]: "boolean" should be lowercase here, it is not a proper noun and the type is expressed as lower case in TypeScript's type system

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 83f3d04f246e453d21bd8ab821f6d84bf838ce12.

**Pull request closed without merging**